README.md

```diff
@@ -29,13 +29,24 @@ In the future, one goal is to put all dependencies as optional, using different
 
 ## How to contribute?
 
-Just fork the repository and send us a Pull Request (PR) using the `develop` branch as a point of start for your contribution.
-
-Each PR need to work only in one issue or in a [single responsibility](https://en.wikipedia.org/wiki/Single_responsibility_principle).
-
-Please, name your PR using the same style than others already closed on this project.
-
-Make sure your branch builds without any warnings/issues.
+Follow these steps:
+
+1. Fork this project (Click on `Fork` bottom)
+2. Clone your forked project to your local machine (`git clone
+https://github.com/USERNAME/james.git`)
+3. Open an issue, describing what is the problem / or the enhancement you will
+make. After that, anotate the issue ID (the # next to issue title)
+4. Create a new branch following this pattern: `pr#ISSUE-ID` (`git checkout -b
```

Comment:
@fabriciofx please change to `issue#ISSUE-ID` and say this is only a suggestion.